app/javascript/components/public_health/CustomExport.js

```diff
@@ -236,23 +236,42 @@ class CustomExport extends React.Component {
                       {this.props.patient_query.filter?.map((filter, index) => {
                         return (
                           <Badge key={`filter-${index}`} variant="primary" className="mr-1">
```

Comment:
noticing that this badge is not wrapping if it's too big (see screenshot)

<img width="786" alt="Screen Shot 2021-07-08 at 2 42 47 PM" src="https://user-images.githubusercontent.com/35042815/124974483-d9926480-dffa-11eb-85cb-2bad200dedf6.png">

Reply:
would it make more sense to split each of them out into their own badges?

Reply:
I wonder if there is a good way to abbreviate the multi ones so the badge is a reasonable length... like maybe not include all the information?

Reply:
yeah I thought about maybe only displaying the values and not the field names, but that could be confusing for labs where there's a report and specimen collection date

Reply:
the changes look AWESOME!